File: problems/algorithms/string-generating-cpf/js/response/generating-cpf.print.mjs
import { cpfCheckDigit } from './generating-cpf.js';

// Generating CPF Check Digit

// invalid cpf
console.log(cpfCheckDigit('000000000'));
console.log(false);

// the check digit of 123456789 is 09
console.log(cpfCheckDigit('123456789'));
console.log('12345678909');

// the check digit of 000111222 is 85
console.log(cpfCheckDigit('000111222'));
console.log('00011122285');

File: src/pages/exercises/string-generating-cpf/_codes/ruby/response/generating-cpf.print.rb
require_relative 'generating-cpf.rb'

# Generating CPF Check Digit

# invalid cpf
puts cpf_check_digit('000000000') 
puts false 

# the check digit of 123456789 is 09
puts cpf_check_digit('123456789') 
puts '12345678909' 

# the check digit of 000111222 is 85
puts cpf_check_digit('000111222') 
puts '00011122285' 
